app/models/game.rb

```diff
@@ -10,6 +11,11 @@ class Game < ApplicationRecord
                         .or(Game.where('player_1_id IS NULL AND player_2_id IS NOT NULL'))
                     }
 
+  def render_piece(x, y)
+    piece = pieces.where(x_coordinate: x).where(y_coordinate: y)[0]
+    piece.nil? ? '' : piece.icon
```

Comment:
```
return nil if piece.nil?

piece.icon
```

I find `nil` is more clear than and empty string.

Then in frontend you could use `.nil?`